cmd/notary/integration_test.go

```diff
@@ -2125,7 +2131,7 @@ func TestClientKeyImport(t *testing.T) {
 	// if there is hardware available, root will only be on hardware, and not
```

Comment:
These 8 tests could go into 8 separate testing functions but the setup for each is common at the top so not sure if we would like to repeat that across testing functions. CC @david regarding our conversation IRL earlier today